lib/_internal/_words/_words.test.js

```diff
@@ -0,0 +1,15 @@
+import _words from '.'
+
+it('Returns words in a string', () => {
+  const phrase = 'You only lose when you give up.'
+  const words = _words(phrase)
+
+  expect(words.length).toEqual(8)
```

Comment:
Am I missing something or why does it return 8 instead of 7 ?

Reply:
It looks like the issue of regex . See `replace(/([A-Z]+)/g, '_$1')`, Currently it also replaces the first letter of string which it shouldn't

So `_words('You only lose when you give up.')` => `[ '', 'You', 'only', 'lose', 'when', 'you', 'give', 'up.' ]`